internal/autocert/certmagic_logger.go

```diff
@@ -0,0 +1,50 @@
+package autocert
+
+import (
+	"strings"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/pomerium/pomerium/internal/log"
+)
+
+type certMagicLoggerCore struct {
+	core   zapcore.Core
+	fields []zapcore.Field
+}
+
+func (c certMagicLoggerCore) Enabled(lvl zapcore.Level) bool {
+	return c.core.Enabled(lvl)
+}
+
+func (c certMagicLoggerCore) With(fs []zapcore.Field) zapcore.Core {
+	return certMagicLoggerCore{core: c.core, fields: append(c.fields, fs...)}
+}
+
+func (c certMagicLoggerCore) Check(e zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
+	return ce.AddCore(e, c)
```

Comment:
Should this be conditional based on a call to `c.Enabled(e.Level)`, in order to respect the wrapped core's log level?

(I'm a little confused by the `Core` interface, so I might be misunderstanding the contract; please let me know if that's the case.)